ui/pages/onboarding-flow/create-password/create-password.js

```diff
@@ -58,18 +62,77 @@ export default function CreatePassword({
   }, [password, confirmPassword, passwordError, confirmPasswordError]);
 
   const handlePasswordChange = (passwordInput) => {
-    let error = '';
     let confirmError = '';
-    if (passwordInput && passwordInput.length < 8) {
-      error = t('passwordNotLongEnough');
+    let passwordStrengthDescription = '';
+    let passwordStrengthInput;
+    const passwordEvaluation = zxcvbn(passwordInput);
+
+    switch (passwordEvaluation.score) {
```

Comment:
Since the only two things that change here are class name and text, it seems like we can refactor this a bit. What do you think about this?

``` javascript
// You could define these outside of the component since they don't depend on anything, if you wanted

function getPasswordStrengthLabel(score, t) {
  if (score >= 4) {
    return {
      className: "create-password__form__strong",
      text: t("strong")
    }
  } else if (score === 3) {
    return {
      className: "create-password__form__average",
      text: t("average")
    };
  } else {
    return {
      className: "create-password__form__weak",
      text: t("weak")
    }
  } 
}

// Later on, inside the component...

const passwordStrengthLabel = getPasswordStrengthLabel(passwordEvaluation.score, t);
const passwordStrengthInput = t('passwordStrength', [
  <span
    key={passwordEvaluation.score}
    className={passwordStrengthLabel.className}
  >
    {passwordStrengthLabel.text}
  </span>,
]);
```

Reply:
@mcmire Fixed in: 31b1950541693058091725c294105bf99a39b4fa